packages/orbit-components/src/PricingTable/__tests__/index.test.js

```diff
@@ -1,188 +1,104 @@
     );
-  });
-  it("should have actions", () => {
-    expect(pricingTableItem.find("Button").exists()).toBe(true);
-  });
-  it("shouldn't have Radio button", () => {
-    expect(pricingTableItem.find("Radio").exists()).toBe(false);
-  });
-  it("should be actionable", () => {
-    pricingTableItem.simulate("click");
-    expect(onClick2).toHaveBeenCalled();
+
+    expect(screen.getByRole("button")).toBeInTheDocument();
+    expect(screen.getByText("kek")).toBeInTheDocument();
   });
 });
 
 describe("PricingTable", () => {
-  const mobileDescription = "Basic ticket fare includes:";
-  const component = mount(
-    <PricingTable activeElement={0} dataTest={dataTest}>
-      <PricingTableItem
-        dataTest={dataTest}
-        name={name}
-        price={price}
-        featureIcon={image}
-        badge={badge}
-        action={actions}
-        active
-        onClick={onClick2}
-        mobileDescription={mobileDescription}
-      >
-        {content}
-      </PricingTableItem>
-      <PricingTableItem
-        dataTest={dataTest}
-        name={name}
-        price={price}
-        featureIcon={image}
-        badge={badge}
-        action={actions}
-        active
-        onClick={onClick2}
-      >
-        {content}
-      </PricingTableItem>
-      <PricingTableItem
-        dataTest={dataTest}
-        name={name}
-        price={price}
-        featureIcon={image}
-        badge={badge}
-        action={actions}
-        active
-        onClick={onClick2}
-      >
-        {content}
-      </PricingTableItem>
-    </PricingTable>,
-  );
-
-  it("should render data-test", () => {
-    expect(component.render().prop("data-test")).toBe(dataTest);
-  });
-  it("should render children", () => {
-    expect(component.find("PricingTableItem")).toHaveLength(4);
-  });
-  it("should render childrens actions", () => {
-    expect(component.find("Button").exists()).toBe(true);
-  });
-  it("should render childrens children", () => {
-    expect(component.find("ul").exists()).toBe(true);
+  it("should have expected DOM output", () => {
+    useMediaQueryMock.mockImplementation(() => ({ isDesktop: true }));
+
+    render(
+      <PricingTable activeElement={0} dataTest="test">
+        <PricingTableItem dataTest="first" active>
+          kek
+        </PricingTableItem>
+        <PricingTableItem dataTest="second">bur</PricingTableItem>
+      </PricingTable>,
+    );
+
+    const trimBoxShadow = theme.orbit.boxShadowAction
+      .replace(/\s(?![^)]*\()/g, "")
+      .split(", ")
+      .join();
+
+    const boxShadowActive = trimBoxShadow.concat(
+      `,inset 0 0 0 2px ${theme.orbit.paletteBlueNormal}`,
+    );
+
+    expect(screen.getByTestId("test")).toBeInTheDocument();
+    expect(screen.getByText("kek")).toBeInTheDocument();
+    expect(screen.getByTestId("first")).toHaveStyleRule("box-shadow", boxShadowActive);
+    expect(screen.getByTestId("second")).toHaveStyleRule("box-shadow", trimBoxShadow);
```

Comment:
Box shadow is eye candy that could change later, breaking our test. A broken test should generally indicate a broken functionality, so tests should be written to be resilient. Unit tests should test only critical features, styles are mostly for visual regression tests.